frb_example/pure_dart/rust/src/api.rs

```diff
@@ -1277,6 +1277,13 @@ pub fn frb_generator_test() -> RustOpaque<FrbOpaqueReturn> {
     panic!("dummy code");
 }
 
+/// Structure for testing the SyncReturn<RustOpaque> code generator.
+/// FrbOpaqueSyncReturn must be only return type.
+/// FrbOpaqueSyncReturn must should be without wrapper like Option<> Vec<> etc.
+pub fn frb_sync_generator_test() -> SyncReturn<RustOpaque<FrbOpaqueSyncReturn>> {
```

Comment:
well, we should add a few lines in the rust side to check it does return what we want

Reply:
I did not quite understand. Can you explain in more detail? This is a test for the code generator to generate the necessary functions and finalizer

Reply:
Yes I agree. But without anything in the dart side https://github.com/fzyzcjy/flutter_rust_bridge/blob/master/frb_example/pure_dart/dart/lib/main.dart, we do not know whether, for example, those generated code panics

Reply:
In theory, we know when we run dart tests. If no methods are generated dart will just crash.
This is exactly how I found this error.

Reply:
Yes, the question is, does it work *in runtime*. For example, will it happen that, when calling this function, it never returns the desired rustopaque, but somehow panic? (I know it won't - but would be great to test it)

just as simple as:

```
test('...'), () {
  await api.frb_sync_generator_test();
});

Reply:
no problem
```
expect(instanceof(api.frbSyncGeneratorTest(), FrbOpaqueSyncReturn), isTrue);
```